.gitignore

```diff
@@ -108,3 +108,15 @@ website/static/js/*
 !website/static/js/mathjax.js
 !website/static/js/code_block_buttons.js
 website/static/_sphinx-sources/
+
+# JMG
+*.aux
+*.pygtex
+/jmg_notes/*.aux
+/jmg_notes/*.log
+/jmg_notes/*.out
+/jmg_notes/*.gz
+/jmg_notes/*.toc
+/jmg_notes/*.w18
+/jmg_notes/*.dvi
+/jmg_notes/*.ps
```

Comment:
Can you remove the these from the .gitignore? You can commit only the relevant files by doing

```
git add compute_dp_sgd_privacy.py README.md
git commit -m "commit message"
```

Reply:
Done, for .gitignore